Command/ExpiredVideoNotificationCommand.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Pumukit\ExpiredVideoBundle\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+
+class ExpiredVideoNotificationCommand extends ContainerAwareCommand
+{
+    private $dm = null;
+    private $mmobjRepo = null;
+    private $type = "expired";
+    private $user_code = 'owner';
+    private $factoryService;
+    private $logger;
+
+    protected function configure()
+    {
+        $this
+            ->setName('video:expired:notification')
+            ->setDescription('Automatically sending notifications to users who have a video about to expire.')
+            ->addArgument('days', InputArgument::REQUIRED, 'days')
+            ->setHelp(
+                <<<EOT
+Automatic email sending to owners who have videos that expire soon
+
+Arguments: 
+    days : Videos that expire in the next X days
+EOT
+            );
+    }
+
+    private function initParameters()
+    {
+        $this->dm = $this->getContainer()->get('doctrine_mongodb')->getManager();
+        $this->expiredVideoService = $this->getContainer()->get('pumukit_expired_video.notification');
+
+        $this->mmobjRepo = $this->dm->getRepository('PumukitSchemaBundle:MultimediaObject');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->initParameters();
+        $days = intval($input->getArgument('days'));
+        if (!is_int($days)) {
+            $output->writeln('Please, write an integer number');
+        }
+
+        $mmObj = $this->findExpiredVideos($days);
+
+        $this->getOwnerEmails($output, $mmObj);
+
+        return;
+    }
+
+    private function findExpiredVideos($days)
+    {
+        $this->dm = $this->getContainer()->get('doctrine_mongodb')->getManager();
+        $mmobjExpired = $this->getExpiredVideos($days);
+
+        return $mmobjExpired;
+    }
+
+    private function getOwnerEmails(OutputInterface $output, $mmobjExpired)
```

Comment:
@Yurujai. maybe the name of this function should be different as it doesn't return the owner emails, instead, it gets the owners emails and send notification, it doesn't return any value.